packages/commcare/test/index.js

```diff
@@ -226,9 +221,12 @@ describe('getCases', () => {
       .intercept({
         path: `/a/${domain}/api/v0.5/case`,
         method: 'GET',
+        query: {
```

Comment:
I can't really tell where this is in the diff, but should this test have changed?